main.rb

```diff
@@ -1,8 +1,48 @@
-require_relative './app'
+require './app'
+
+class Run
+  def initialize
+    @run = App.new
+  end
+
+  def menu
+    choices = ['1 - List all Books',
+               '2 - List all Music Albums',
+               '3 - List all Games',
+               '4 - List all Genres',
+               '5 - List all labels',
+               '6 - List all Authors',
+               '7 - Add a book',
```

Comment:
- [ ] Well, while adding a new item, you should also handle errors as well. For example, if I add a wrong value you need to display an error message. {`Your entity is incorrect...`} something like that. When I tried to enter a wrong value it shows this:

<img width="900" alt="Screen Shot 2022-08-18 at 23 48 10" src="https://user-images.githubusercontent.com/69011963/185472044-885898ce-7f25-4575-a3c9-42a0f50da5e7.png">